pl_examples/basic_examples/mnist_examples/image_classifier_2_lite.py

```diff
@@ -0,0 +1,157 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Here are 5 required steps to convert to LightningLite.
+
+1. Subclass LightningLite and override its ``run`` method.
+
+2. Move the body of your existing ``run`` function into the ``run`` method.
+
+3. Remove all ``.to``, ``.cuda`` etc calls since LightningLite will take care of it.
+
+4. Apply ``setup`` over each model and optimizers pair, ``setup_dataloaders`` on all your dataloaders,
+and replace ``loss.backward()`` with ``self.backward(loss)``.
+
+5. Instantiate your LightningLite and call its ``run`` method.
+
+Learn more from the documentation: https://pytorch-lightning.readthedocs.io/en/latest/starter/lightning_lite.html.
+"""
+
+import argparse
+
+import torch
+import torch.nn.functional as F
+import torch.optim as optim
+import torchvision.transforms as T
+from torch.optim.lr_scheduler import StepLR
+from torchmetrics.classification import Accuracy
+
+from pl_examples.basic_examples.mnist_datamodule import MNIST
+from pl_examples.basic_examples.mnist_examples.image_classifier_1_pytorch import Net
+from pytorch_lightning import seed_everything
+from pytorch_lightning.lite import LightningLite  # import LightningLite
+
+
+class Lite(LightningLite):
+    def run(self, hparams):
```

Comment:
add typing? and why not as
```suggestion
    def run(self, **hparams):
```

Reply:
Because hparams here is a argparse Namespace.
Later on in example 3-5, self.hparams will become the LightningModule.hparams.

Reply:
Because it is similar to the action of save_hyperparameters